### ISSUE TITLE
1.7.1: -execute stopped working with terminal-notifier as Alert in Notifications

### ISSUE THREAD

**Issue by JayBrown** [julienXX/terminal-notifier#176]
I have terminal-notifier set as an "Alert" in System Preferences > Notifications

I.e. it is visible until I click "Close" or "Show", and in the latter case I sometimes execute commands, e.g.

```
-execute "open -a TextEdit /path/to/tempfile.txt && open /path/to/app.app"
```

"Close" still works in 1.7.1, but clicking "Show" does not run the execute commands anymore.

Don't know if this is a bug, or if there's still a different t-n behavior I have to take into account. (?)

**Comment by julienXX** [comment]
Hi, it looks like a bug. Adding `-timeout 10`, `-reply` or `-actions` options makes it work in the meantime.

**Comment by JayBrown** [comment]
It's just in one or two of my scripts, so I think I can wait it out until the bug fix. No rush. Do the more important stuff first. :)

**Comment by tewe** [comment]
Hit by this on 10.11.6.

None of the workaround options work.

Might be #181

**Comment by JayBrown** [comment]
The weird thing that it _does_ work sometimes, but only in 1 out of 20 or 30 or so, so I kept trying. For example, I have a background script looking for updates of my installed homebrew formulas. If there are updates, I get a notification. (It's a standard script floating around here on GitHub with some minor tweaks.) If it's an important update I run it, but manually with a command from my own BitBar system menulet. Furthermore, I added a little notification function if the homebrew stdout tells me that there are new formulas; in that case, when clicking "Show", terminal-notifier should open a temp txt file with the `brew` stdout and the application **Cakebrew**. In a lot of cases this doesn't work, but I don't know what the reason is. Maybe it has something to do with the background script having already exited or still running? (But it has only happened with the latest update.)

(I'm on 10.11.6 as well, SIP disabled; switching to Sierra as soon as a decent version of GPGMail is available.)
